layers/+lang/python/packages.el

```diff
@@ -420,3 +422,20 @@ fix this issue."
                  (eq 'yapf python-formatter))
         (add-hook 'python-mode-hook 'yapf-mode)))
     :config (spacemacs|hide-lighter yapf-mode)))
+
+;;https://github.com/andrew-christianson/lsp-python-ms
+;;Not on MELPA
```

Comment:
Should this comment be removed?
@yyoncho Mentioned in a previous comment that it's on melpa now.

Reply:
Yep - had updated the layer to use the melpa version, but forgotten to strip the comment. Thanks